src/aura/STG_CMP_Tabs/STG_CMP_Tabs.cmp

```diff
@@ -38,6 +38,23 @@
     <aura:handler event="c:STG_EVT_Save" action="{!c.saveSettings}" />
     <aura:handler event="c:STG_EVT_Cancel" action="{!c.resetSettings}" />
 
+    <div class="slds-hide" id="successToast" aura:id="successToast">
+        <div class="slds-box">
+            <div class="slds-notify_container slds-notify_container--inline">
+                <div class="slds-notify slds-notify--toast slds-theme--success" role="alert">
+                    <span class="slds-assistive-text">Success</span>
+                    <button class="slds-button slds-button--icon-inverse slds-notify__close" tabindex="-1">
+                        <c:svgIcon name="close" category="utility" size="small" assistiveText="Close" pressIcon="{!c.clostToast}" />
```

Comment:
@lifewithryan When I tried clicking the X, nothing happened right away. I think maybe it's because the pressIcon should be c.closeToast? (spelling error)

Reply:
that's odd...I remember fixing this locally, maybe I didn't commit the change :/